build-script.sh

```diff
@@ -44,31 +57,29 @@ for dir in $DIRS; do
     V1_10="1.10"
     V1_11="1.11"
     V1_12="1.12"
+    V1_13="1.13"
+    V1_14="1.14"
     if [[ "$VERSION" =~ .*"$V1_7".* || "$VERSION" =~ .*"$V1_8".* ]]; then
       echo "Not installing GRPC for old versions"
-    elif [[ "$VERSION" =~ .*"$V1_9" || "$VERSION" =~ .*"$V1_10" || "$VERSION" =~ .*"$V1_11" || "$VERSION" =~ .*"$V1_12" ]]; then
+    elif [[ "$VERSION" =~ .*"$V1_9" || "$VERSION" =~ .*"$V1_10" || "$VERSION" =~ .*"$V1_11" || "$VERSION" =~ .*"$V1_12" ||  "$VERSION" =~ .*"$V1_13" || "$VERSION" =~ .*"$V1_14" ]]; then
       # install v3 dependencies that support this go version
-      set +e
-      go get -u google.golang.org/grpc # this go get will fail to build
-      set -e
-      cd $GOPATH/src/google.golang.org/grpc
-      git checkout v1.31.0
-      cd -
-
-      set +e
-      go get -u golang.org/x/net/http2 # this go get will fail to build
-      set -e
-      cd $GOPATH/src/golang.org/x/net/http2
-      git checkout 7fd8e65b642006927f6cec5cb4241df7f98a2210
-      cd -
+      pin_go_dependency "google.golang.org/grpc@v1.31.0"
```

Comment:
Love the way you generalized the whole pinning mechanism to make this so much easier to read.

Reply:
I basically made a simple bash version of `go get` 🤣